packages/rx_bloc_cli/lib/src/models/generator_arguments_provider.dart

```diff
@@ -77,7 +77,11 @@ class GeneratorArgumentsProvider {
     // OTP
     final otpEnabled = _reader.read<bool>(CommandArguments.otp);
 
-    if (otpEnabled && !(loginEnabled || socialLoginsEnabled)) {
+    // Pin Code
+    final pinCodeEnabled = _reader.read<bool>(CommandArguments.pinCode);
+
+    if (otpEnabled &&
+        !(loginEnabled || socialLoginsEnabled || pinCodeEnabled)) {
```

Comment:
I think the correct condition here would be: if we have OTP or PIN enabled `(otpEnabled || pinEnabled)` and we don't have any login `!(loginEnabled || socialLoginsEnabled)`, we should enable the login due to OTP / PIN requirement.

Reply:
Ok, this was changed.